server/api/documents.js

```diff
@@ -7,16 +7,37 @@ router.get(
   "/",
   passport.authenticate("jwt", { session: false }),
   (req, res) => {
-    docsDb
-      .getAllDocuments()
-      .then(data => {
-        res.send(data);
-      })
-      .catch(err => {
-        console.error(err);
-        res.send(500);
-      });
+    let user = req.user.role;
```

Comment:
This should be defined as a `const` if it's not going to change. Moreover, the naming is incorrect. This should be named `role` and not `user`, it's a bit confusing otherwise.

Reply:
it is changed to const